kernel/component-manager/src/main/java/org/sakaiproject/util/SakaiApplicationContext.java

```diff
@@ -34,7 +34,7 @@
  * PostProcessor objects (e.g., SakaiProperties) a chance to do their work,
  * and load a few central components before the rest.
  */
-public class SakaiApplicationContext extends GenericApplicationContext {
+public class SakaiApplicationContext extends GenericWebApplicationContext {
```

Comment:
This was introduced to help Spring Boot and Sakai relate each other.

java.lang.IllegalArgumentException: Unable to call initializer. Object of class [org.sakaiproject.util.SakaiApplicationContext] must be an instance of interface org.springframework.web.context.ConfigurableWebApplicationContext

NOTE: GenericWebApplicationContext  extends GenericApplicationContext